content/articles/broadcasting-a-pirate-fm-radio-station-using-a-raspberry-pi/index.md

```diff
@@ -0,0 +1,105 @@
+---
+layout: engineering-education
+status: 
+published: 
+url: /broadcasting-a-pirate-fm-radio-station-using-a-raspberry-pi/
+title: Broadcasting a private FM radio using Raspberry Pi
+description: This article will explain how one can create a private FM radio with the use of a Raspberry Pi and internet connection. The private FM radio broadcast can reach a radius of around fifty meters.
+author: ruphus-muita
+date: 2021-07-19T00:00:00-18:00
+topics: [Networking]
+excerpt_separator: <!--more-->
+images:
+
+  - url: /engineering-education/broadcasting-a-pirate-fm-radio-station-using-a-raspberry-pi/hero.jpg
+    alt: Raspberry Pi Image 
+---
```

Comment:
Hi @ruphus-muita this was unnecessary. Please don't add it next time. It's called 'front matter' and it's the job of a peer reviewer to add them. As peer reviewers, we also add a 'Peer Reviewers Contributors (PRC) line' at the bottom of the article. You'll see it once I'm done reviewing your article. Don't add the two. You'll get the hang of it with time. Don't worry.

Reply:
@Bayler Thanks for the insight. This is done.